index.html

```diff
@@ -0,0 +1,15 @@
+<html>
```

Comment:
2 things

- can you format this file with vscode
- in vscode enter `!` so we can get a `DOCTYPE` in the file and some other stuff, see the below ~1 min video for an example
https://www.youtube.com/watch?v=4Dfra3FXDNM